pkg/generic/httpthrift_codec.go

```diff
@@ -46,15 +46,29 @@ type HTTPRequest = descriptor.HTTPRequest
 type HTTPResponse = descriptor.HTTPResponse
 
 type httpThriftCodec struct {
-	svcDsc           atomic.Value // *idl
-	provider         DescriptorProvider
-	codec            remote.PayloadCodec
-	binaryWithBase64 bool
+	svcDsc              atomic.Value // *idl
+	provider            DescriptorProvider
+	codec               remote.PayloadCodec
+	binaryWithBase64    bool
+	enableDynamicgoReq  bool
+	enableDynamicgoResp bool
+	convOpts            conv.Options
```

Comment:
convOpts is only used for dynamicgo, rename as `dynamicgoConvOpts`, it is better for code reading

Reply:
no need this code anymore